rizza/entity_tester.py

```diff
@@ -198,6 +203,39 @@ def execute(self, mock=False):
             logger.debug('fail: {}'.format(handled))
             return {'fail': handled}
 
+    def form_input_decapsulation(self, **kwargs):
+        if isinstance(kwargs['inpt'], list):
+            if not kwargs['inpt']:
+                return []
+            lst = []
+            for inpt in kwargs['inpt']:
+                if isinstance(inpt, list):
+                    lst.append([
+                        self.form_input_decapsulation(
+                        field=kwargs['field'],
+                        inpt=inpt,
+                        imeths=kwargs['imeths'],
+                        )
+                    ])
+                else:
+                    if inpt == 'url':
+                        print(inpt)
+                    lst.append(form_input(
+                        inpt,
+                        kwargs['imeths'],
+                        kwargs['field'],
+                        self.config
+                    ))
+            return lst
+        # TODO dict
```

Comment:
the same needs to be done for dict, 
and of course dict of lists and lists of dict.